dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -16,25 +16,31 @@
 			device_type = "cpu";
 			compatible = "cdns,tensilica-xtensa-lx7";
 			reg = <0>;
-			cpu-power-states = <&off>;
+			cpu-power-states = <&suspend &off>;
 		};
 
 		cpu1: cpu@1 {
 			device_type = "cpu";
 			compatible = "cdns,tensilica-xtensa-lx7";
 			reg = <1>;
-			cpu-power-states = <&off>;
+			cpu-power-states = <&suspend &off>;
 		};
 
 		cpu2: cpu@2 {
 			device_type = "cpu";
 			compatible = "cdns,tensilica-xtensa-lx7";
 			reg = <2>;
-			cpu-power-states = <&off>;
+			cpu-power-states = <&suspend &off>;
 		};
 	};
 
 	power-states {
+		suspend: suspend {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-idle";
```

Comment:
ACE D0i3 power state does not match suspend-to-idle and it can be confusing. The recommendation is to use substate_id set to ACE defined D0i3 value.

Reply:
Introducing substates currently is not necessary. I have change states name to better reflect those defined in documentation.